src/components/TodoInfo/TodoInfo.tsx

```diff
@@ -1,4 +1,29 @@
-// Add the required types and props
-export const TodoInfo = () => (
-  <>TodoInfo markup</>
-);
+import classNames from 'classnames';
+import { Todo } from '../../types/Todo';
+import { UserInfo } from '../UserInfo/UserInfo';
+
+type Props = {
+  todo: Todo;
+};
+
+export const TodoInfo: React.FC<Props> = ({ todo }) => {
+  const {
+    title,
+    user,
+    completed,
+  } = todo;
+
+  return (
+    <>
+      <article className={classNames(
+        'TodoInfo',
+        { 'TodoInfo--completed': completed },
+      )}
+      >
+        <h2 className="TodoInfo__title">{title}</h2>
+
+        {user && <UserInfo user={user} />}
+      </article>
+    </>
```

Comment:
it's a redundant tag wrapper
```suggestion
      <article className={classNames(
        'TodoInfo',
        { 'TodoInfo--completed': completed },
      )}
      >
        <h2 className="TodoInfo__title">{title}</h2>

        {user && <UserInfo user={user} />}
      </article>
```